### ISSUE TITLE
Compile a questionnaire on deformation model usage

### ISSUE THREAD

**Issue by ccrook** [opengeospatial/CRS-Deformation-Models#1]
This issue will be closed at the end of 17th August.  Please offer comments/suggestions for improvement before then.

This issue is to design a questionnaire to collect information about current and planned usage of deformation models for coordinate transformation.  The questionnaire has two parts.  The first seeks general information about the usage of deformation models in a jurisdiction.  The second seeks information about each specific model that is planned or in use.  This is primarily to inform the project team on the minimum specifications for a functional model that will serve the current and expected requirements.  However it may also provide a good reference data set on the status of deformation models.

The proposed survey questionnaire is as follows:
______

### Deformation/velocity model survey

This questionnaire is seeking information about actual or planned usage of deformation models published
to align data sets observed at different epochs.   This does not include geophysical models constructed
to analyse deformation unless they are directly used for coordinate transformations.

??? more clarity on what a deformation model is.…

This survey comes from the Open Geospatial Consortium project team tasked with developing a functional
model for deformation models.  Its purpose is to inform the project team of the nature of deformation models
that are in use or planned.  

The questionnaire is in two sections.  The first section seeks general information on the usage of deformation 
models, and the second section is asking for detailed information about each deformation model in use or 
planned - please provide a separate response for each model.  Not all the questions may be applicable for every model.

Thank you very much for taking time to provide this information and support this initiative.  

**General information on deformation model usage**

* Country/jurisdiction 
* Agency: 
* Contact details:
* What deformation models are currently in use?
* What deformation models are planned?
* What are the business drivers to implement a deformation model? (Who will benefit from being able to use it, and how?):
* What is inhibiting implementing or using deformation models (eg legislation):
* What types of users already use or do you expect to use deformation models
* In what software and systems do users need the deformation model to be implemented?

**Details of specific models** 

(Please answer for each model separately)

* Name of model
* Country/jurisdiction
* Agency
* Contact
* Documentation (eg url, reference to paper)
* What is the status of the model? (planned, in development, active, historic)
* What is the dimension of the deformation? (horizontal/vertical/3d)
* Describe the approximate spatial extent of model (regional, continental, project) ?
* How is the model defined spatially? (eg grid, triangulated network)
* What spatial interpolation method is used? (eg bilinear interpolation)
* What time behaviour(s) are used by the model (eg constant velocity, step function, exponential function)?
* How is uncertainty of the deformation calculated?
* What geodetic or vertical datum (or datums) is the model associated with? 
* How is the model used (coordinate system transformation, point motion model)
* Is the deformation model maintained, how often
* What is the expected lifetime of the model
* How is the model provided to users (eg  online service, downloadable url)? 
* What format or formats is the model supplied in?
* What accuracy do users require from the model (relative local accuracy, accuracy relative to global datums)
* Does the model meet user requirements? If not what more is needed?
* What sources of deformation are implemented in the model? (eg secular tectonic deformation, earthquakes)
* How is the deformation model derived (what data and models are used, eg CORS time series, InSAR, geophysical fault models)
* How is it used with active control networks (eg CORS, real time positioning services)

**Comment by ccrook** [comment]
Kevin Kelly.  Should be structured as a question (with a question mark?)

**Comment by palmerj** [comment]
I wonder if we can refine the "Current and expected users of deformation models" question to be "Current and expected users of deformation models, and their use cases".  That way you can get more details about how they need to use it. e.g Desktop Software or embedded field device for xxx

**Comment by ccrook** [comment]
@palmerj wrote:

> I wonder if we can refine the "Current and expected users of deformation models" question to be "Current and expected users of deformation models, and their use cases"

I've proposed a slightly different question "In what software and systems do users need the deformation model to be implemented?" It is not quite the same question - do you think it would get the information you want.  It could be something like "What the users use cases that require or would benefit from the deformation model?" , but I'm not sure how widely that would be understood outside the development community.  What do you think?

**Comment by palmerj** [comment]
> "In what software and systems do users need the deformation model to be implemented?"

This is good.

> "What the users use cases that require or would benefit from the deformation model?"

This is not required. However, it's triggered another thought. Do you think the Business drivers question covers (or is explicit enough) to tease out the end user value for implementing the model? Maybe we should be clearer here as well.

**Comment by palmerj** [comment]
> Spatial definition of deformation: Grid/TIN/other? Interpolation method.
> Time models: velocity function, step function, other

Should we also be asking if the model that the jurisdiction has implemented needs all the features it has? Sometimes the more complex modelling is not required given the data resolution or required accuracy.

**Comment by ccrook** [comment]
> Should we also be asking if the model that the jurisdiction has implemented needs all the features it has? Sometimes the more complex modelling is not required given the data resolution or required accuracy.

I've added 

> * Does the model meet requirements? If not what more is needed?

to try to capture that thought.

**Comment by ccrook** [comment]
> Do you think the Business drivers question covers (or is explicit enough) to tease out the end user value for implementing the model? Maybe we should be clearer here as well.

How about...

> * What are the business drivers to implement a deformation model?  Who will benefit from being able to use it, and how?

**Comment by palmerj** [comment]
Fantastic!

**Comment by ccrook** [comment]
@kevinmkelly As you suggested at the meeting on 13th I have reworded the questionnaire as questions.  I've also reordered them and added an introduction to provide some context to the questions.

**Comment by ccrook** [comment]
From the 10 Aug 2020 meeting chat:

Dana Caccamise (NOAA's NGS): 21:40: I suggest that we remove survey from the title and suggest "Defromation/ Velocity Model Usage Questionare"

**Comment by ElmarBrockmann** [comment]
Possible additional question:
- in addition to "In what software and systems do users need the deformation model to be implemented?": which software tools and data sets are provided to users (self-developed programs, data files in specific formats, web tools, api's, dll's, implemenations directly in GIS software or GNSS receivers)?
- in addtion to "Is the deformation model maintained, how often": Are there rules how to use updated deformation models or is backward-compartibility an issue?

**Comment by kbevers** [comment]
> What is inhibiting implementing or using deformation models (eg legislation):

In addition to this I would like to explicitly ask about what license or terms of use the model is available under. There are several reasons for that:

1. Raise awareness of putting a license on data and models provided to the public. Many agencies don't state the terms of use of their models which has the unfortunate consequence that they can't be used in open source software (we need explicit permission).
2. We may want to use a particular model in a prototype - we need to know if we are allowed to do that or not.

The question could be frased something like:

"What license or terms of use, if any, is the model released under"

**Comment by ccrook** [comment]
@kbevers: Good thought on licensing - I've added your suggestion verbatim.

@ElmarBrockmann: Also good point - I draughted the questionnaire on the assumption of models being published, but some models are not actually published - just exposed in custom software or services at the moment.   I've added a question under the section about specific deformation models

* What software or services are provided to support usage of the model?

Do you think this captures what you were asking?

**Comment by ccrook** [comment]
It also occurs to me that the introductory notes should say what will happen with the data that is collected beyond informing our work, and in particular whether it will be publicly available.

My thinking is it should be made public.  Does anyone think that would inhibit potential responders?

**Comment by kevinmkelly** [comment]
@ccrook: Are we collecting or requesting data? Do we need to collect data? We are interested in deformation models in use or development and lots of background information about them. I don't think we need the model data, do we? If we are seeking data, this _may_ put off some potential respondents. 

@kbevers: If we want a sample model and data, perhaps someone from this study group can provide one (or more). We may be getting ahead of ourselves if we start talking about incensing and publication of data at this stage. That would come when we are ready to draft a real OGC Standard. So far, this is an "information" gathering working group.

**Comment by kbevers** [comment]
@kevinmkelly I fail to see how asking about terms of use of a model fall outside the category "information gathering".

**Comment by kevinmkelly** [comment]
@kbevers: You are correct, I got a bit carried away. I'm mainly objecting to what I perceived to imply that we will be collecting data from respondents. If models exist and are in use, surely we can inquire about their terms of use.

**Comment by ccrook** [comment]
@kevinmkelly.  Sorry - the use of the word data has caused some confusion.  By data I just meant the answer to the questions themselves.  I certainly wasn't asking for the models themselves.  Just if anyone is answering these questions they may want to know what happens to the answers, whether they will be able to see other answers and vice versa!

**Comment by kevinmkelly** [comment]
@ccrook: Thanks for clarifying Chris. Sorry for the misunderstanding!

**Comment by ccrook** [comment]
I have created a first cut Google Forms questionnaire at 

https://docs.google.com/forms/d/11PCSVojPPD062P96veEjuqSKjnRMCrU8TyIruWCbWt0/viewform?edit_requested=true

Comments please?

**Comment by ElmarBrockmann** [comment]
Chris - very nice proposal! Very logic questions, good structure.
Just some remarks to:

- inhibiting: might be possible reasons could also be:
   * lack of data determining deformation models
   * no significant movements compared to coordinate reference frame accuracy

- Users: might be more potential answers
  * cadastral work
  * only high precision users

 - very last question (How is it used with active control networks...) - might be moved up to the "User" question adressed above.

- naming of the questionnair "OGC deformation/velocity model":
 overlapps with the other discussion on names. I'm fine with the current naming of the questionnair. Should IAG adressed already in the name because this abbreviation might be known at mapping agency level?   
 Might be, we could ask in the last page on the details of the deformation model:
     * Which naming convention do you use in your country if any is in place? Deformation model? Velocity model?  Point motion model?
    * Any preference concerning naming if not yet introduced?

**Comment by rstanaway** [comment]
Hi Chris,
That looks fine to me and I've already submitted a sample response for Papua New Guinea to see if it worked. I'll also circulate this amongst the IAG WG community so you may wish to include the IAG in the title as Elmar has suggested. There's no real benefit doubling up efforts with parallel WG that have similar motivations.

On that note, Elmar, would this questionnaire also work with your EUREF WG colleagues?

**Comment by ElmarBrockmann** [comment]
@rstanaway - Yes, certainly. In any cause we also can discuss it in the EUREF Governing board.  A questionnaire released also to the directors of the mapping agencies (which forward it then to the reference frame specialist to give an answer) could also be an option making them aware of activities in this direction. But, currently, only the Nordic countries have a deformation model in use.

**Comment by rstanaway** [comment]
@ElmarBrockmann - That's great. I was going to get in touch with Martin Lidberg and Pasi Haekli as well. Do you know who the coordinator is of the deformation model used in Iceland? Also, who would be the best contact for any models used in Italy?

**Comment by kbevers** [comment]
> Do you know who the coordinator is of the deformation model used in Iceland?

@rstanaway That would be Guðmundur Valsson.

**Comment by ElmarBrockmann** [comment]
@rstanaway - Italy and Greece have several universities working on the topic. Mapping agencies have other contact persons. So, difficult to answer. In the EUREF "EU Dense Velocity" working group we try to invite them all - and they joined. So, on the technical level, at least 2-3 groups per mentioned country are involved already. For the "deformation model" part the situation is more unclear.
Martin is current chair of EUREF and involved in all nordic activities. Pasi is the person behind the most recent nordic uplift model - he kindly provided a pre-next-release-version which I was able to include last month to the above mentioned EUREF WG (-> http://pnac.swisstopo.admin.ch/divers/dens_vel/index.html).

**Comment by ccrook** [comment]
@ElmarBrockmann  Sorry about slow response to your comments on the questionnaire, I've added most of your comments.  Also added one other question for any other comments.  I didn't add a question about what to call the deformation model as I think that it makes sense to have that in context of a discussion about terminology and how it fits with standard terminology rather than based on what we currently call such models.  But I did add a question for any other comments.  We may in any case get some ideas from the "Name of the model" question.  Also I didn't add IAG to the title - I guess this is currently being driven from OGC, and also contextually it is about deformation model in the context of coordinate transfoormations rather than in a scientific geodesy context.  The last question on deformation models in relation to active control networks was proposed by @rstanaway - I read it as more a question about how it is or might be used rather than who is using it, so I have left it as is.
